tests/helixprep.proj

```diff
@@ -74,15 +74,15 @@
     Condition="'$(TargetsWindows)' == 'true' ">
 
     <PropertyGroup>
-      <CrossgenVar Condition="'$(Crossgen)' == 'true'">set RunCrossGen=true</CrossgenVar>
+      <CmdCrossgenVar Condition="'$(Crossgen)' == 'true'">set RunCrossGen=true</CrossgenVar>
```

Comment:
@gkhanna79 we don't necessarily, I just thought it'd be better to specify that it was only going to be used in the CMD file.